include/shark/Models/Trees/CARTClassifier.h

```diff
@@ -100,7 +100,7 @@ class CARTClassifier : public AbstractModel<RealVector,LabelType>
 	/// Constructor taking the splitMatrix as argument
 	CARTClassifier(SplitMatrixType const& splitMatrix)
 	{
-		setSplitMatrix(splitMatrix);
+		m_splitMatrix=splitMatrix;
```

Comment:
Yes, that's because when loading a model that was exported using this code, the splitMatrix is already optimized, and optimizeSplitMatrix was not needed. I experienced a big bottleneck when loading a model using setSplitMatrix. (Sorry, I did this long time ago, I added a misleading line in the PR description).
But I understand, to be safer it's better to stay with `setSplitMatrix(splitMatrix)`.
Since I use it [here](https://github.com/elehcim/Shark/commit/24b0a186682e83594ccd1226fa70188a945eef03#diff-6811b3d4bf9cbf971149c0ba3342dd73R137), maybe I can add an `operator=` without optimizeSplitMatrix and leave this constructor as is.